src/com/main/db/Database.java

```diff
@@ -30,7 +36,119 @@ public void dbClose() {
 			e.printStackTrace();
 		}
 	}
+	
+	//JAMES - ITEM
+	/*
+	 * @parameters
+	 * i -> an item that has just been created and now is being inserted into the database
+	 */
+	public void addItem(Item i) {
+		dbConnect();
+		String stmt = "insert into item(vendorId,name,price,quantity)values (?,?,?,?)";
+		try {
+			PreparedStatement p = con.prepareStatement(stmt);
+			p.setInt(1, i.getVendorId());
+			p.setString(2, i.getName());
+			p.setFloat(3, i.getPrice());
+			p.setInt(4, i.getQuantity());
+			p.executeUpdate();
+			System.out.println("Item successfully added to your menu!!!");
+		}catch(SQLException f) {
+			f.printStackTrace();
+		}
+		dbClose();
+	}
+	
+	
+	/*James
+	 * 
+	 * @parameters
+	 * name --> is the name of an item that the vendor has 
+	 * vendorId --> is the vendor identification number
+	 * 
+	 * the purpose of this function is to validate that the is an item with name "name" that belongs 
+	 * to vendorId
+	 * 
+	 * 
+	 * ********this function is no longer used as of the creation of fetchItems********
```

Comment:
Do we still need this function if its not being used anymore? Maybe comment the function out

Reply:
there is other stuff I can change that is small too, I will do it tomorrow